testing/integration/fqdn_test.go

```diff
@@ -103,8 +103,14 @@ func (s *FQDN) TestFQDN() {
 			kibana.MonitoringEnabledLogs,
 			kibana.MonitoringEnabledMetrics,
 		},
+		AgentFeatures: []map[string]interface{}{
+			{
+				"name":    "fqdn",
+				"enabled": false,
+			},
+		},
 	}
-	policy, err := tools.InstallAgentWithPolicy(s.T(), s.agentFixture, kibClient, createPolicyReq)
+	policy, err := tools.InstallAgentWithPolicy(s.T(), s.agentFixture, kibClient, s.requirementsInfo.ESClient, createPolicyReq)
```

Comment:
It looks like this change broke the build on `main`:

https://buildkite.com/elastic/elastic-agent/builds/1394#01890d53-55b0-4225-8786-35bb175ef164/56-241

Once https://github.com/elastic/elastic-agent/pull/2928 is merged, CI will be able to catch such issues on PRs before they're merged into `main`.